zk.js/src/newUtxo.ts

```diff
@@ -327,7 +327,7 @@ export async function encryptOutUtxoInternal({
   utxoHash: Uint8Array;
 }): Promise<Uint8Array> {
   if (encryptionPublicKey) {
-    const ciphertext = Account.encryptNaclUtxo(
+    const ciphertext = hasher.encryptNaclUtxo(
```

Comment:
Maybe we should rename ``hasher``to ``wasmInstance`` it confused me that encrypt function is called from the hasher instance.
wdyt?

Reply:
Yeah, agree, but I think `wasmInstance` is a little bit too generic. I'm thinking about something like `light-wasm`, or `light-rs`, wdyt?